carta/session.py

```diff
@@ -388,10 +388,28 @@ def append_image(self, path, hdu="", complex=False, expression=ArithmeticExpress
         complex : {2}
             Whether the image is complex. Set to ``False`` by default.
         expression : {3}
-            Arithmetic expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ArithmeticExpression.AMPLITUDE`.
+            Complex expression to use if appending a complex-valued image. The default is :obj:`carta.constants.ComplexExpression.AMPLITUDE`.
         """
         return Image.new(self, path, hdu, True, complex, expression)
 
+    @validate(String(), String(), String(r"\d*"), Boolean())
+    def load_LEL_image(self, directory, expression, hdu="", append=False):
+        """Open or append a new image via the Lattice Expression Language (LEL) interface.
+
+        Parameters
+        ----------
+        directory : {0}
```

Comment:
Can full paths be included in the LEL expression? If they are, is this directory even used? Can we make this optional, and default to the file browser starting directory, like we do in the Stokes hypercube function?

Reply:
at the frontend we can do
`"/Users/kswang/set_QA_e2e_v2/dice_one.fits" + "/Users/kswang/carta_image_pool/set_dice/dice_four.fits"`
![Screen Shot 2023-04-26 at 18 26 27](https://user-images.githubusercontent.com/20819712/234548151-52959818-97b9-411d-abc0-4cd6d4e4ff64.png)